src/csharp/Grpc.Core.Api/Grpc.Core.Api.csproj

```diff
@@ -9,10 +9,11 @@
     <PackageTags>gRPC RPC HTTP/2</PackageTags>
     <VersionPrefix>$(GrpcCsharpVersion)</VersionPrefix>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <Nullable>enable</Nullable>
   </PropertyGroup>
 
   <PropertyGroup>
-    <TargetFrameworks>net45;netstandard1.5;netstandard2.0</TargetFrameworks>
```

Comment:
I assume the netstandard2.1 addition has to do with https://github.com/grpc/grpc/pull/27886, not the nullable type attributes?